ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/builder/BuilderApi.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright ConsenSys AG.
```

Comment:
I would not include a file header containing legal notice in the files you created.

Reply:
The gradle build file enforces it. It won't build if it is missing the spdx header